packages/scoutgame/src/builderNfts/recordNftMint.ts

```diff
@@ -288,6 +289,13 @@ export async function recordNftMint(
     log.error('Error recording referral bonus', { error, builderId: builderNft.builderId, scoutId });
   }
 
+  refreshEstimatedPayouts({
```

Comment:
i think this should block, since this is called from the webapp when a user buys an NFT. wouldn't we want the est price to be refreshed when we update the NFT view?

Reply:
200ms on a prod connection, going to await it